secure-storage/secure-storage-impl/src/main/java/com/duckduckgo/securestorage/impl/di/SecureStorageModule.kt

```diff
@@ -41,25 +41,21 @@ object SecureStorageModule {
     fun providesSecureStorageKeyStore(context: Context): SecureStorageKeyStore =
         RealSecureStorageKeyStore(context)
 
-    @Provides
-    fun providesSupportFactory(keyManager: SecureStorageKeyManager): SupportFactory {
-        return SupportFactory(keyManager.l1Key)
-    }
-
     @Provides
     @SingleInstanceIn(AppScope::class)
     fun providesSecureStorageDatabase(
         context: Context,
-        factory: SupportFactory
+        keyManager: SecureStorageKeyManager
     ): SecureStorageDatabase {
         return Room.databaseBuilder(
             context,
             SecureStorageDatabase::class.java,
             "secure_storage_database_encrypted.db"
-        ).openHelperFactory(factory)
+        ).openHelperFactory(SupportFactory(keyManager.l1Key))
             .addMigrations(*ALL_MIGRATIONS)
             .enableMultiInstanceInvalidation()
             .fallbackToDestructiveMigration()
+            .allowMainThreadQueries()
```

Comment:
why do we want this?

Reply:
Ah good catch, we don't want this. I just forgot to remove this since I was using this for testing before. Updated!